README.md

```diff
@@ -37,6 +38,7 @@ You will need the following to build Revit_Toolkit:
     - .NET Framework 4.7.2 for Revit 2020
```

Comment:
Should we change all this to .NET Framework 4.8?

Reply:
good point @vietle-bh - fixed 👍